examples/array_metadata_test.go

```diff
@@ -42,9 +42,9 @@ import (
 	"github.com/TileDB-Inc/TileDB-Go/examples_lib"
 )
 
-// ExampleArrayMetadataArray shows and example creation, writing and reading of a
-// sparse array
-func ExampleArrayMetadataArray() {
+// ExampleRunArrayMetadataArray shows an example of creation, writing, and
+// reading of a sparse array
+func ExampleRunArrayMetadataArray() {
```

Comment:
The changes to the examples look unrelated; can you remove them from the PR?

In a subsequent PR we should fold the `examples_lib` files into `examples`.

Reply:
Yeah, I tried directly calling the functions in `examples` but couldn't for no apparent reason. Such a shame.